apps/webapp/app/presenters/OrganizationsPresenter.server.ts

```diff
@@ -75,16 +82,21 @@ export class OrganizationsPresenter {
 
     //no project in session, let's set one
     if (!sessionProjectId) {
+      //no session id and no project slug so we need to select the best project
       if (!projectSlug) {
-        const bestProject = await this.#selectBestProjectForOrganization(organizationSlug, userId);
+        const bestProject = await this.#selectBestProjectForOrganization(
+          organizationSlug,
+          userId,
+          request
+        );
         const session = await setCurrentProjectId(bestProject.id, request);
         throw redirect(request.url, {
           headers: { "Set-Cookie": await commitCurrentProjectSession(session) },
         });
       }
 
-      //use the project param to find the project
-      const project = await prisma.project.findFirst({
+      //get all the projects
+      const projects = await prisma.project.findMany({
```

Comment:
This allows us to check if there are any projects, as well as find the exact project.